src/DotNetLightning.Core/Serialize/Features.fs

```diff
@@ -173,14 +173,12 @@ module internal Feature =
 /// Uses regular class instead of F# type for caching byte[] representation
```

Comment:
This comment should be deleted, I suppose.

Reply:
Done.